ui/components/ui/box/box.d.ts

```diff
@@ -131,6 +131,14 @@ export type BackgroundColorArray = [
   BackgroundColor?,
 ];
 
+interface BoxDisabledProp {
+  disabled?: boolean;
+}
+
+interface BoxHrefProp {
+  href?: string;
+}
```

Comment:
Hmm Box should accept all html element attributes. If we want to merge this we should create an issue to fix the main issue otherwise we will be creating interface props for every html attribute possible?